### ISSUE TITLE
epoch_acc not defined in census tf-keras example?

### ISSUE THREAD

**Issue by caseyjconger** [GoogleCloudPlatform/cloudml-samples#459]
I think there may be an issue with cloudml-samples/census/tf-keras/hptuning_config.yaml. It seems like the target metric is set to be `epoch_acc` but `epoch_acc` isn't defined anywhere in the code. Should this just be 'accuracy' as that's the metric defined in the model?

**Comment by andrewferlitsch** [comment]
@dizcology PTAL

**Comment by gogasca** [comment]
@caseyjconger for Hyper parameter tuning we use the TensorBoard callback.
https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/TensorBoard what happens is that this callback returns a summary name of epoch_ because of this: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/callbacks.py#L2084
You may need to create custom callbacks and it summary operation to use other names.

**Comment by gogasca** [comment]
Updated sample, please take a look [here](https://github.com/GoogleCloudPlatform/ai-platform-samples/blob/master/training/tensorflow/census/tf-keras/trainer/task.py#L116).
